### ISSUE TITLE
No Logs getting captured after executing apex script

### ISSUE THREAD

**Issue by snathpatil** [joeferraro/MavensMate#751]
v7.0.1:

* **SublimeText**:

v0.0.10

Windows 7 64bit:

No:

34.0:

cs51:

Execute a simple debug statement in a new script OR choose option 'Execute Apex' 

There was a directory debug/logs with few debugs. When I used option 'Flush Debug Logs', no new logs are getting captured there.

Also Apex Script logs are not getting captured under /apex-scripts/log

**Comment by snathpatil** [comment]
I have clicked on option 'Start Logging', yet no logs captured.

**Comment by joeferraro** [comment]
You may need to use v36.0 of the API. The API for creating/managing trace flags has changed over the past few API releases.

**Comment by snathpatil** [comment]
Thanks Joe. After updating API to 36, it is capturing debug now. 

However after executing 'Apex Script', logs are in /debug/logs and not in /apex-scripts/log.

**Comment by joeferraro** [comment]
@snathpatil correct, all logs, (test runs, execute anonymous, etc.) are now streamed directly to /logs.

> Also logs are getting captured from all the users who have set a trace flag, and not just the one I have configured in .debug file!

This certainly should not be happening. Can you please submit a MavensMate debug log? You can configure them in Desktop settings

**Comment by snathpatil** [comment]
@joeferraro please see attached log file. Although I got error 'Error: Could not create Trace Flags: LogType is required.: LogType', debug logs are still getting captured from other users except the user id I set in .debug file.

[mavensmate.zip](https://github.com/joeferraro/MavensMate/files/482833/mavensmate.zip)

**Comment by joeferraro** [comment]
@snathpatil I'm not sure whether this will help, but you may want to try updating to 0.0.11-beta.5 and seeing whether the issue persists.

**Comment by snathpatil** [comment]
@joeferraro after updating to v0.0.11-beta.5, this issue seems to be resolved. Thanks!
